spec/index.md

```diff
@@ -273,8 +275,12 @@ KL27 storage memory layout:
     - default → DATA.BIN
   - File size (from start of data section)  (`0x02`)
     - 4B size in bytes (MSB First)
-    - default → 126 KBs
+    - default → 129024 (126 KBs)
     - max size is 126 KBs (128KB - 1KB for flash interface config and 1KB for DAL's config)
+  - Base64 encoding start address (from start of data section)  (`0x09`)
```

Comment:
Is Base64 accurate here? i thought we were encoding each byte to two hex-digits rather then using Base64? if so, can we just change the description here?

Reply:
sorry @gerargz  - my bad! :)

Reply:
Updated with the content from microbit-foundation/microbit-design@05a25eb